lib/test-collection.js

```diff
@@ -225,6 +226,18 @@ class TestCollection extends EventEmitter {
 		}
 		return false;
 	}
+
+	finalize() {
+		globals.setImmediate(() => {
+			Array.from(this.pendingTestInstances).forEach(test => {
```

Comment:
Use a `for-of` loop instead of `forEach`.

Reply:
Done.